common/src/main/scala/explore/components/ui/PartnerFlags.scala

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.components.ui
+
+import lucuma.core.model.Partner
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSImport
+
+object PartnerFlags {
+  @js.native
+  @JSImport("resources/images/flags/argentina-flag-icon-32.png", JSImport.Default)
```

Comment:
I think this will be fine but we should check after we do full opt if there are any issues

Reply:
I wonder if we should consider using WebP, there maybe some backwards compatibility issues but that could be solved using `picture`
https://www.html5rocks.com/en/tutorials/responsive/picture-element/#toc-file-type

Reply:
I'll take a look at that.